docs/Course Wiki/PHYS Course Offerings/PHYS214.md

```diff
@@ -0,0 +1,54 @@
+# PHYS214
+
+PHYS214 (Quantum Physics) is a 2-credit hour, 8-week long, course that is required for physics majors. It is required for ECE majors. It is offered in the fall, spring, and summer semesters.
```

Comment:
change to "(University Physics: Quantum Physics)", also change to "that is required for ECE majors as a part of the Foundational Mathematics and Science requirement. It is also required for physics majors."  Also mention it is a first 8-weeks course.